activesupport/CHANGELOG.md

```diff
@@ -54,8 +65,6 @@
     "#<ActiveSupport::KeyGenerator:0x0000000104888038>"
     ```
 
-    *Petrik de Heus*
```

Comment:
@p8  did you mean to remove this?

Reply:
Nah, It's me that borked the rebase. I'll fix.

Reply:
https://github.com/rails/rails/pull/48821

Reply:
Fixed in 008cbbd351267cbf5f005f9bb2f9320f654d9873